src/generators/java/presets/JacksonPreset.ts

```diff
@@ -67,6 +68,19 @@ export const JAVA_JACKSON_PRESET: JavaPreset = {
 
       blocks.push(content);
 
+      return renderer.renderBlock(blocks);
+    },
+    getter({ renderer, property, content, model }) {
+      if (model.options.isExtended) {
+        return content;
+      }
+      const isAdditionalProperties =
+        property.propertyName === 'additionalProperties';
+      const blocks: string[] = [];
+      if (isAdditionalProperties) {
+        blocks.push(renderer.renderAnnotation('JsonAnyGetter'));
+      }
```

Comment:
Where did the check for constrained dictionary model with unwrap serialization type go? 🤔

Reply:
I forgot to update this block of code as well. I am guessing I should replace this:
```ts
      const isAdditionalProperties =
        property.propertyName === 'additionalProperties';
      const blocks: string[] = [];
      if (isAdditionalProperties) {
```

with this:
```
      //Properties that are dictionaries with unwrapped options, cannot get the annotation because it cannot be accurately unwrapped by the jackson library.
      const isDictionary =
        property.property instanceof ConstrainedDictionaryModel;
      const hasUnwrappedOptions =
        isDictionary &&
        (property.property as ConstrainedDictionaryModel).serializationType ===
          'unwrap';
      const blocks: string[] = [];
      if (hasUnwrappedOptions) {
```

Reply:
Yep, looks right